### ISSUE TITLE
Add to documentation: how to install Tensorflow Lite onto Raspberry Pi 3B+ Raspian Stretch

### ISSUE THREAD

**Issue by jdlamstein** [tensorflow/tensorflow#19110]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No.
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Raspberry Pi 3B+, Raspian Stretch
- **TensorFlow installed from (source or binary)**:
What I'm asking about. 
- **TensorFlow version (use command below)**:
Tensorflow Version 1.7.0
- **Python version**: 
Python 3.5
- **Bazel version (if compiling from source)**:
What I'm asking about. 
- **GCC/Compiler version (if compiling from source)**:
What I'm asking about. 
- **CUDA/cuDNN version**:
N/A
- **GPU model and memory**:
Broadcom VideoCore IV @ 250 MHz (BCM2837: 3D part of GPU @ 300 MHz, video part of GPU @ 400 MHz)
- **Exact command to reproduce**:
Addition to documentation: Installation of Tensorflow Lite on Raspberry Pi 3B+ Raspian Stretch. 

### Describe the problem
I would like instructions on how to install Tensorflow Lite onto Raspberry Pi 3B+ with Raspbian Stretch OS. To the best of my knowledge, the documentation doesn't yet cover installation for Raspbian Stretch. I posted on the Raspberry Pi Stack Exchange (https://raspberrypi.stackexchange.com/questions/83498/how-do-i-install-tensorflow-lite-on-raspbian-stretch) and I was directed here. My goal is to deploy a Tensorflow neural network onto the Raspberry Pi 3B+ with Tensorflow Lite.

**Comment by raygeeknyc** [comment]
I’ve got a project using the tf nightly for Pi, thanks for that, with the Object Detection API but inference time is ~1.5 secs; I exorctvthat I could use tf lite instead and experience significantly better performance.  Any ETA for RPi support?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @petewarden: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @petewarden: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @petewarden: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @petewarden: It has been 59 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tofulawrence** [comment]
Resolved with nightly. For latency issue, please use quantized model, which is fast.

**Comment by jdlamstein** [comment]
@tofulawrence @petewarden @achowdhery 
Thank you!

**Comment by arvindpdmn** [comment]
Documentation is here: https://www.tensorflow.org/lite/guide/build_rpi
